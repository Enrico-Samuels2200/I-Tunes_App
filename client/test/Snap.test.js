import React from 'react';  
import Discover from '../routes/iTunesDiscover.js';  
import renderer from 'react-test-renderer';

test('renders correctly', () => {    
    let com = renderer.create(<Discover/>)     
    let tree = com.toJSON();   
    expect(tree).toMatchSnapshot();  
})