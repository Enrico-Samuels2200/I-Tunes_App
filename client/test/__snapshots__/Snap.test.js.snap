// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="navBar"
  >
    <h1>
      I Tunes Search Engine
    </h1>
    <input
      autoComplete="off"
      name="artistName"
      onChange={[Function]}
      placeholder="Enter title here"
    />
    <select
      className="typePick"
      onChange={[Function]}
    >
       
      <option
        value="all"
      >
        all
      </option>
      <option
        value="music"
      >
        music
      </option>
      <option
        value="audio book"
      >
        audio book
      </option>
      <option
        value="ebook"
      >
        ebook
      </option>
    </select>
    <button
      onClick={[Function]}
    >
      Search
    </button>
    <br />
  </div>
  <hr />
</div>
`;
